src/dronevis/models/gesture_recognition.py

```diff
@@ -0,0 +1,202 @@
+            while True:
+                _, frame = cap.read()
+                cur_time = time.time()
+                image = self.predict(frame)
+                fps = 1 / (cur_time - prev_time)
+                cv2.imshow(window_name, write_fps(image, fps))
+                prev_time = cur_time
+                if cv2.waitKey(1) & 0xFF == ord("q"):
+                    break
+        cv2.destroyAllWindows()
+        cap.release()
+
+
+    def __calc_landmark_list(self,image: np.ndarray, landmarks:dict)-> list:
+        """calculate the landwarks in an image
+
+        Args:
+            image (np.array): input image 
+            landmarks (dict): dictonary of hand keypoints detected in an image.
+
+        Returns:
+            list: list of hand keypoints detected in an image
+        """
+        image_width, image_height = image.shape[1], image.shape[0]
+        landmark_point = []
+        for _, landmark in enumerate(landmarks.landmark):
+            landmark_x = min(int(landmark.x * image_width), image_width - 1)
+            landmark_y = min(int(landmark.y * image_height), image_height - 1)
+            landmark_z = landmark.z
+            landmark_point.append([landmark_x, landmark_y,landmark_z])
+        return landmark_point
+
+    def __pre_process_landmark(self,landmark_list:list)->list:
+        """converts calculated landmarks to relative coordinates and normalizes them
+
+        Args:
+            image (np.array): input image 
+            landmark_list (dict): list of hand keypoints detected in an image.
+
+        Returns:
+            list: list of normalized hand keypoints after conversion to relative coordinates
+        """
+        temp_landmark_list = copy.deepcopy(landmark_list)
+        base_x, base_y, base_z = 0, 0, 0
+        for index, landmark_point in enumerate(temp_landmark_list):
+            if index == 0:
+                base_x, base_y, base_z = landmark_point[0], landmark_point[1], landmark_point[2]
+
+            temp_landmark_list[index][0] = temp_landmark_list[index][0] - base_x
+            temp_landmark_list[index][1] = temp_landmark_list[index][1] - base_y
+            temp_landmark_list[index][2] = temp_landmark_list[index][2] - base_z
+        
+        temp_landmark_list = list(
+            itertools.chain.from_iterable(temp_landmark_list))
+
+        max_value = max(list(map(abs, temp_landmark_list)))
+
+        def normalize_(n):
+            return n / max_value
+
+        temp_landmark_list = list(map(normalize_, temp_landmark_list))
+
+        return temp_landmark_list
+    
+
+
+class Keypoint_classifier(nn.Module):
+    def __init__(self):
+        super(Keypoint_classifier, self).__init__()
+        self.fc1 =  nn.Linear(63, 50)
+        torch.nn.init.xavier_uniform_(self.fc1.weight)
+        self.leakyrelu = nn.LeakyReLU(inplace=True)
+        self.fc2 =  nn.Linear(50, 6)
+        torch.nn.init.xavier_uniform_(self.fc2.weight)
+              
+    def forward(self, x):
```

Comment:
The argument `x` is not compliant with PEP8 with 2 prospectives. 
(1) It is not written in snake_case form, i.e. minimum length of a variable is 3.
(2) It doesn't have a datatype (I guess its datatype should be `torch.Tensor`).

One last thing is that the method is missing a docstring.